salsa_dancing_molecules/eqiulibrium.py

```diff
@@ -0,0 +1,28 @@
+"""Module for finding the "steady state" of a system."""
+
+from pymbar.timeseries import detect_equilibration
+import numpy as np
+from asap3 import Trajectory
+
+
+def get_eqiulibrium(traj, ensemble='NVE'):
+    """
+    Find the eqiulibrium of a system.
+
+    Input:
+        list - array, containing parameters for each timestep
```

Comment:
You have used `traj: ase.io.trajectory.Trajectory  - traj file containing atom objects` as the description for the traj argument in the other file. Is this the same kind of object?